Extractor/ExposedRoutesExtractor.php

```diff
@@ -58,7 +58,7 @@ class ExposedRoutesExtractor implements ExposedRoutesExtractorInterface
      *
      * @throws \Exception
      */
-    public function __construct(RouterInterface $router, array $routesToExpose = array(), $cacheDir, $bundles = array())
+    public function __construct(RouterInterface $router, array $routesToExpose, $cacheDir, $bundles = array())
```

Comment:
💡 This line triggered a deprecation error on php 8:
```
Required parameter $cacheDir follows optional parameter $routesToExpose
```
We can safely remove the default value because constructing this class without providing a value for  `$routesToExpose` was never possible.